WordPress/wordpress-playground
# wp-now: add `--wp` parameter to change WordPress versions

- Closes https://github.com/WordPress/wordpress-playground/issues/228

## Proposed changes:

- `wp-now` accepts the parameter `--wp` to change the WordPress version.

Example of the full command:
```sh
nx preview wp-now start --path=/path/to/your-theme-or-plugin --wp=6.0
```

This command will check if that version exists in your home directory `~/.wp-now/wordpress-versions/6.0` and if not it will download it.
Currently the [`sqlite-database-integration`](https://wordpress.org/plugins/sqlite-database-integration/) plugin supports WordPress version from 6.0+

If the WordPress version doesn't exist it will display the following error message:
`WordPress 6.33 not found. Check https://wordpress.org/download/releases/ for available versions.`

## Testing instructions

- Build packages `nvm use && yarn install && yarn build`
- Run `nx preview wp-now start --path=/path/to/your-theme-or-plugin --wp=6.0` or any other valid version.
- Observe `~/.wp-now/wordpress-versions/6.0` exist and WordPress has started correctly.
